serialization/compact/src/main/java/CompactZeroConfig.java

```diff
@@ -0,0 +1,32 @@
+import com.hazelcast.config.CompactSerializationConfig;
+import com.hazelcast.config.Config;
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.core.HazelcastInstance;
+
+import java.util.Map;
+
+/**
+ * Demonstrates how to use Compact(BETA) serialization with java only setup
+ * If all hazelcast clients and members are java, then you can use Compact format without any config.
+ */
+public class CompactZeroConfig {
+
+    public static void main(String[] args) {
+        Config config = new Config();
+        //This config is needed only during BETA phase.
+        CompactSerializationConfig compactSerializationConfig = config.getSerializationConfig().getCompactSerializationConfig();
+        compactSerializationConfig.setEnabled(true);
+        //Note that PersonDTO does not implement Serializable. In such case, following config should be added
```

Comment:
This detail may be confusing in this example. May be you can add another example just for this.

Reply:
I am having trouble naming the examples. Since this is one line addition, I felt like this is a good point to add.

Reply:
Having a separate example would make this distinction more visible to the user but I am also OK with this then.